lib/coverband/configuration.rb

```diff
@@ -12,6 +12,20 @@ class Configuration
     attr_writer :logger, :s3_region, :s3_bucket, :s3_access_key_id, :s3_secret_access_key
     attr_reader :track_gems, :ignore, :use_oneshot_lines_coverage
 
+    #####
+    # TODO: This is is brittle and not a great solution to avoid deploy time
+    # actions polluting the 'runtime' metrics
+    #
+    # * should we skip /bin/rails webpacker:compile ?
+    # * Perhaps detect heroku deployment ENV var opposed to tasks?
+    #####
+    IGNORE_TASKS = ['assets:clean',
```

Comment:
I don't think we need `assets:clean` or `assets:precompile` anymore. Ideally we are only ignoring only `coverband:` namespaced tasks.

Reply:
I will test that on coverband demo and see if we can remove

Reply:
Ok sounds good. If we do need it, it would be a good opportunity to add a test to expose that need if we don't have one already.

Reply:
yes no longer needed removing.